pkg/reconciler/brokercell/resources/deployments.go

```diff
@@ -64,13 +64,16 @@ func MakeIngressDeployment(args IngressArgs) *appsv1.Deployment {
 	}
 	container.Resources = corev1.ResourceRequirements{
 		Limits: corev1.ResourceList{
-			corev1.ResourceMemory: resource.MustParse("1000Mi"),
+			corev1.ResourceMemory: resource.MustParse(args.MemoryLimit),
```

Comment:
Yes. However, this won't happen due to defaulting logic, where memory limit is defaulted based on memory request, if not explicitly set.

Reply:
I think it's better to code defensively here. We may change the logic of defaulting and forget about updating this. Also try avoid using `MustParse`. Previously it's ok because we hardcoded the values and we know they are valid. Now the values could be specified by users, and from what I can tell we don't do properly validation in the webhook.